### PR TITLE
Add suport for ATmega88p

### PR DESCRIPTION
**Pull request by blueberry-fan**
This adds support for ATmega88p. 

This depends on PR https://github.com/Rahix/avr-device/pull/166, though. I didn't update corresponding dependencies in cargo.toml yet.

### CODE REVIEW

**Comment by stappersg** [comment]
On Sun, Dec 01, 2024 at 02:23:53AM -0800, blueberry-fan wrote:
> This adds support for ATmega88p. 
> 
> This depends on PR https://github.com/Rahix/avr-device/pull/166, though.

That one got merged.


> I didn't update corresponding dependencies in cargo.toml yet.

Somehow I read "it is work in progress".
Please express explicite the current state.
A simple "waits on me" or "now complete" is fine.


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by blueberry-fan** [comment]
Sorry, maybe I was too eager with this PR.
Since there is no new tag for avr-device yet, this is really pending.
Or would it be acceptable to specify a git commit as dependency?

**Comment by tones111** [comment]
#618 should provide the `avr-device` version update you're dependent on, assuming @Rahix determines it to be a minor bump prior to it merging.

**Comment by Rahix** [comment]
@blueberry-fan, please rebase on latest `main` to pull in the new version of `avr-device`. Also add ATmega88P to the CI workflow so we can see whether everything builds as expected :)

**Comment by blueberry-fan** [comment]
I have tried to update as requested. As this was my first contact with githubs's CI, please tell me, If I missed / messed up something.

**Comment by blueberry-fan** [comment]
Thanks for accepting
